openlibrary/templates/lib/nav_head.html

```diff
@@ -35,17 +35,17 @@
           <label class="search-facet-selector">
             <span aria-hidden="true" class="search-facet-value"></span>
             <select aria-label="Search by">
-              <option>All</option>
-              <option>Title</option>
-              <option>Author</option>
-              <option>Subject</option>
-              <option>Advanced</option>
-              <!-- <option>Lists</option> -->
+              <option>$_("All")</option>
+              <option>$_("Title")</option>
+              <option>$_("Author")</option>
+              <option>$_("Subject")</option>
+              <option>$_("Advanced")</option>
+              <!-- <option>$_("Lists")</option> -->
```

Comment:
We can probably remove this commented line. It belongs in the `feature/list-search` branch